src/docs/product/integrations/integration-platform/ui-components/alert-rule-action.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Alert Rule Action
+sidebar_order: 2
+description: "Learn about the alert rule action UI component."
+---
+
+The alert rule action component allows you to get parameters to define routing or configuration in alert rules for your service. When alert rules are triggered for a user, the configured service will receive [issue alert](/product/integrations/integration-platform/webhooks/#issue-alerts) and [metric alert](/product/integrations/integration-platform/webhooks/#metric-alerts) webhook events, with the specified settings for that service, to further route the alert.
```

Comment:
We try to avoid the word "get" in this way in docs, but I'm not sure of a good replacement. I've made a suggestion below: 

```suggestion
The alert rule action component gives you access to parameters to define routing or configuration in alert rules for your service. When alert rules are triggered for a user, the configured service will receive [issue alert](/product/integrations/integration-platform/webhooks/#issue-alerts) and [metric alert](/product/integrations/integration-platform/webhooks/#metric-alerts) webhook events, with the specified settings for that service, to further route the alert.
```